src/System Application/App/File System/src/Lookup/FilePaginationData.Codeunit.al

```diff
@@ -0,0 +1,33 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.FileSystem;
+
+codeunit 9456 "File Pagination Data"
```

Comment:
This is a public codeunit so it should follow the facade pattern and be documented.